tests/helpers.ts

```diff
@@ -200,16 +201,66 @@ export async function fastVerify(fixtureName: string) {
     const expectedResult = fse.readJSONSync(outputPath);
 
     Object.keys(result).forEach((fPath) => {
-      const lines = (result[fPath] || '').split(/\r?\n/);
-      const expectedLines = (expectedResult[fPath] || '').split(/\r?\n/);
+      if (fPath.endsWith('package.json')) {
```

Comment:
I fear we're doing too much in here :thinking: 

Better we get the test suite green for now though.

Any ideas on how to do less, yet test more, welcome. <3
I'll noodle on this

Reply:
I assume package.json is most unstable thing for now, agree about if-else cases, but I could not offer better ways for now, I think we need to see more usecases and errors to figure out better approach here..